x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/pages/rule_creation/helpers.ts

```diff
@@ -451,6 +466,7 @@ export const formatDefineStepData = (defineStepData: DefineStepRule): DefineStep
         threat_filters: ruleFields.threatQueryBar?.filters,
         threat_mapping: ruleFields.threatMapping,
         threat_language: ruleFields.threatQueryBar?.query?.language,
+        ...alertSuppressionFields,
```

Comment:
I've noticed that `formatDefineStepData` sometimes includes `alert_suppression`, whereas `DefineStepRuleJson` does not contain these alert_suppression fields.

Reply:
Good catch, looks like this property was missed from the earlier suppression implementations.
I have added type definitions for it

Reply:
Thanks for adding it!